style/checkstyle-header1.txt

```diff
@@ -14,3 +14,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
Will adding this new line here affect checkstyle? Previously, even without this line break, checkstyle would still prompt for an additional line break.

Reply:
Since the spotless plugin executes earlier than the checkstyle plugin, so checkstyle can be passed correctly.